README.md

```diff
@@ -8,40 +7,44 @@ distinguishes itself with the following features:
 - Totally transparent field encryption: the value will only be encrypted in the database, never in the value
 - Unit testing
 
+## Integrations
+
+The package supports the following integrations:
+
+- Laravel
+
+## Upgrading
+
+If you're upgrading from a previous version you can find some help with that in [the upgrading guide](UPGRADING.md).
+
 ## Installation
-Add `51systems/doctrine-encrypt` to your Composer manifest.
 
-```js
-{
-    "require": {
-        "51systems/doctrine-encrypt": "~5.0"
-    }
-}
+```bash
+composer require 51systems/doctrine-encrypt
 ```
 
 ## Configuration
-### Using ZF2
-Check out the doctrine-encrypt-module at https://github.com/51systems/doctrine-encrypt-module
 
-### Manually
-Add the event subscriber to your entity manager's event manager. Assuming `$em` is your configured entity manager:
+### Laravel
```

Comment:
Indeed, most probably needs splitting out due to the wasteful framework dependency. Alternatively, integration can be mocked out, but that's high risk.